Makefile

```diff
@@ -110,6 +109,18 @@ buildarm: export CGO_ENABLED=0
 buildarm: export GO111MODULE=on
 buildarm: basebuild
 
+buildtest: export GOOS=linux
+buildtest: export GOARCH=amd64
+buildtest: export CGO_ENABLED=0
+buildtest: export GO111MODULE=on
+buildtest: export LDFLAGS=-ldflags "-X github.com/theotw/natssync/pkg.VERSION=${BUILD_VERSION}"
+buildtest: export COVERPKG=-coverpkg=./pkg/...
+buildtest:
+	go test ${LDFLAGS} ${COVERPKG} -c -o out/bridge_client_${GOARCH}_${GOOS}.test tests/apps/bridge_client_test.go
+	go test ${LDFLAGS} ${COVERPKG} -c -o out/bridge_server_${GOARCH}_${GOOS}.test tests/apps/bridge_server_test.go
+	go test ${LDFLAGS} ${COVERPKG} -c -o out/http_proxylet_${GOARCH}_${GOOS}.test tests/apps/http_proxylet_test.go
+	go test ${LDFLAGS} ${COVERPKG} -c -o out/httpproxy_server_${GOARCH}_${GOOS}.test tests/apps/httpproxy_server_test.go
```

Comment:
The `-c` option allows us to build test binaries ahead of time which is useful for speedier deployments down the road since the test container's CMD/entrypoint doesn't need to spend time compiling stuff.